components/interop/interop-service/src/main/kotlin/net/corda/interop/service/impl/RemoteAliasLookUpServiceImpl.kt

```diff
@@ -0,0 +1,34 @@
+package net.corda.interop.service.impl
+
+import net.corda.sandbox.type.UsedByFlow
+import net.corda.v5.application.interop.AliasMemberInfo
+import net.corda.v5.application.interop.InteropGroupInfo
+import net.corda.v5.application.interop.RemoteAliasLookUpService
+import net.corda.v5.serialization.SingletonSerializeAsToken
+import org.osgi.service.component.annotations.Activate
+import org.osgi.service.component.annotations.Component
+import org.osgi.service.component.annotations.ServiceScope.PROTOTYPE
+
+@Component(service = [RemoteAliasLookUpService::class, UsedByFlow::class], scope = PROTOTYPE)
+class RemoteAliasLookUpServiceImpl @Activate constructor() : RemoteAliasLookUpService, UsedByFlow,
+    SingletonSerializeAsToken {
```

Comment:
```suggestion
class RemoteAliasLookUpServiceImpl @Activate constructor() :
    RemoteAliasLookUpService, UsedByFlow, SingletonSerializeAsToken {
```
I think it's more readable to have all the inherited in a single newline.

Reply:
Fixed in 7c369e5